src/core/state.go

```diff
@@ -23,6 +23,21 @@ import (
 	"github.com/thought-machine/please/src/process"
 )
 
+type ParseMode uint8
+
+const (
+	ParseModeForSubinclude ParseMode = 1 << iota
```

Comment:
how about adding a `ParseModeNormal = 0` constant? We often call things with 0 which is a little unclear at the call site.